app/views/posts/show.html.erb

```diff
@@ -11,18 +11,27 @@
           <span class="card-text">Likes: <%= @post.likes_counter%></span>
         </div>
       </div>
-      <h5>Post Title: <%= @post.title %> </h5>
+      <h5>Post Title: <%= @post.title %></h5>
       <p class="card-text"><%= @post.text %></p>
+      <% if @post.author.id == current_user.id || current_user.admin? %>
```

Comment:
- **[OPTIONAL]** It would be nice to use the `can?` method as that is part of the tools `cancancan` provides.